### ISSUE TITLE
Crash caused by PagedMediathekShowListAdapter

### ISSUE THREAD

**Issue by cemrich** [mediathekview/zapp#347]
```
java.lang.IllegalStateException: Unknown view
at de.christinecoenen.code.zapp.app.mediathek.ui.list.adapter.PagedMediathekShowListAdapter.getItemViewType(PagedMediathekShowListAdapter.kt:35)
at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:608)
at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6)
                                                      getViewForPosition
at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:54)
at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1)
at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:54)
at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:842)
at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:85)
at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:72)
at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:8)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:67)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:155)
at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:1)
at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:43)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1829)
at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1673)
at android.widget.LinearLayout.onLayout(LinearLayout.java:1582)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
at com.android.internal.policy.DecorView.onLayout(DecorView.java:1418)
at android.view.View.layout(View.java:24958)
at android.view.ViewGroup.layout(ViewGroup.java:6784)
at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:4787)
at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:4115)
at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2991)
at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:10665)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1301)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1309)
at android.view.Choreographer.doCallbacks(Choreographer.java:923)
at android.view.Choreographer.doFrame(Choreographer.java:852)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1283)
at android.os.Handler.handleCallback(Handler.java:942)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:226)
at android.os.Looper.loop(Looper.java:313)
at android.app.ActivityThread.main(ActivityThread.java:8741)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
```

Version 8.0.0, Android 13 - I don't know yet, what caused it.

**Comment by cemrich** [comment]
We may have to disable placeholders in PagingConfig.

**Comment by cemrich** [comment]
This is fixed in 8.2.0

**Comment by WebworkrNet** [comment]
Thanks for fixing it. 
The error no longer occurs. :-)
